--- /mnt/data/2021_11_multi_chunk_repair/APRConfig/results_defects4j2/defects4j2_JacksonCore_21/defects4j2_JacksonCore_21/src/main/java/com/fasterxml/jackson/core/filter/FilteringParserDelegate.java	2022-05-03 07:23:09.516886985 -0400
+++ /mnt/data/2021_11_multi_chunk_repair/APRConfig/results_defects4j2/defects4j2_JacksonCore_21/mcrepair_pfl/patch/susp_0/donor_0/patch_0/FilteringParserDelegate.java	2022-05-03 07:25:11.708698599 -0400
@@ -237,7 +237,7 @@
             //if not scalar and ended successfully, and !includePath, then return null
                 if (_currToken.isStructEnd()) {
                     if (_headContext.isStartHandled()) {
-                        return (_currToken = null);
+                        
                     }
                 } else if (_currToken.isScalarValue()) {
                     //else if scalar, and scalar not present in obj/array and !includePath and INCLUDE_ALL matched once

