--- /mnt/data/2021_11_multi_chunk_repair/APRConfig/results_defects4j2/defects4j2_Csv_9/defects4j2_Csv_9/src/main/java/org/apache/commons/csv/CSVRecord.java	2022-05-03 06:02:04.257094951 -0400
+++ /mnt/data/2021_11_multi_chunk_repair/APRConfig/results_defects4j2/defects4j2_Csv_9/mcrepair_pfl/patch/susp_0/donor_3/patch_0/CSVRecord.java	2022-05-03 06:04:04.910463953 -0400
@@ -177,6 +177,10 @@
      * @return the given map.
      */
     <M extends Map<String, String>> M putIn(final M map) {
+if (mapping == null) {
+return map;
+}
+
         for (final Entry<String, Integer> entry : mapping.entrySet()) {
             final int col = entry.getValue().intValue();
             if (col < values.length) {

