--- /mnt/data/2021_11_multi_chunk_repair/APRConfig/results_defects4j2/defects4j2_Compress_1/defects4j2_Compress_1/src/main/java/org/apache/commons/compress/archivers/cpio/CpioArchiveOutputStream.java	2022-05-03 07:19:57.830429986 -0400
+++ /mnt/data/2021_11_multi_chunk_repair/APRConfig/results_defects4j2/defects4j2_Compress_1/mcrepair_pfl/patch/susp_0/donor_4/patch_8/CpioArchiveOutputStream.java	2022-05-03 07:20:56.748857264 -0400
@@ -342,6 +342,7 @@
      *             occurred
      */
     public void close() throws IOException {
+finish();
         if (!this.closed) {
             super.close();
             this.closed = true;

