--- /mnt/data/2021_11_multi_chunk_repair/APRConfig/results_defects4j2/defects4j2_Cli_14/defects4j2_Cli_14/src/java/org/apache/commons/cli2/option/GroupImpl.java	2022-05-02 18:51:47.475519403 -0400
+++ /mnt/data/2021_11_multi_chunk_repair/APRConfig/results_defects4j2/defects4j2_Cli_14/mcrepair_pfl/patch/susp_0/donor_1/patch_1/GroupImpl.java	2022-05-02 18:53:31.867282917 -0400
@@ -259,7 +259,7 @@
                     break;
                 }
 
-                option.validate(commandLine);
+                
             }
         }
 

