--- /mnt/data/2021_11_multi_chunk_repair/APRConfig/results_defects4j2/defects4j2_Compress_24/defects4j2_Compress_24/src/main/java/org/apache/commons/compress/archivers/tar/TarUtils.java	2022-05-03 06:41:42.395859550 -0400
+++ /mnt/data/2021_11_multi_chunk_repair/APRConfig/results_defects4j2/defects4j2_Compress_24/mcrepair_pfl/patch/susp_0/donor_0/patch_10/TarUtils.java	2022-05-03 06:46:17.251181676 -0400
@@ -129,8 +129,7 @@
         if (trailer == 0 || trailer == ' '){
             end--;
         } else {
-            throw new IllegalArgumentException(
-                    exceptionMessage(buffer, offset, length, end-1, trailer));
+            
         }
         trailer = buffer[end - 1];
         while (start < end - 1 && (trailer == 0 || trailer == ' ')) {

