--- /mnt/data/2021_11_multi_chunk_repair/APRConfig/results_defects4j2/defects4j2_Cli_19/defects4j2_Cli_19/src/java/org/apache/commons/cli/PosixParser.java	2022-05-03 03:15:52.881770581 -0400
+++ /mnt/data/2021_11_multi_chunk_repair/APRConfig/results_defects4j2/defects4j2_Cli_19/mcrepair_pfl/patch/susp_0/donor_28/patch_2/PosixParser.java	2022-05-03 03:21:47.106538768 -0400
@@ -231,7 +231,7 @@
             currentOption = options.getOption(token);
             tokens.add(token);
         }
-        else if (stopAtNonOption)
+        else if (!eatTheRest)
         {
             eatTheRest = true;
             tokens.add(token);

