--- /mnt/data/2021_11_multi_chunk_repair/APRConfig/results_defects4j2/defects4j2_Cli_14/defects4j2_Cli_14/src/java/org/apache/commons/cli2/option/GroupImpl.java	2022-05-02 19:02:58.667717111 -0400
+++ /mnt/data/2021_11_multi_chunk_repair/APRConfig/results_defects4j2/defects4j2_Cli_14/mcrepair_sfl/patch/susp_8/donor_1/patch_1/GroupImpl.java	2022-05-02 19:55:43.089779258 -0400
@@ -259,7 +259,7 @@
                     break;
                 }
 
-                option.validate(commandLine);
+                
             }
         }
 

