--- /mnt/data/2021_11_multi_chunk_repair/APRConfig/results_defects4j2/defects4j2_Compress_31/defects4j2_Compress_31/src/main/java/org/apache/commons/compress/archivers/tar/TarUtils.java	2022-05-03 23:21:25.177907558 -0400
+++ /mnt/data/2021_11_multi_chunk_repair/APRConfig/results_defects4j2/defects4j2_Compress_31/mcrepair_pfl/patch/susp_0/donor_0/patch_10/TarUtils.java	2022-05-03 23:29:29.281831439 -0400
@@ -132,9 +132,7 @@
 
         for ( ;start < end; start++) {
             final byte currentByte = buffer[start];
-            if (currentByte == 0) {
-                break;
-            }
+            
             // CheckStyle:MagicNumber OFF
             if (currentByte < '0' || currentByte > '7'){
                 throw new IllegalArgumentException(

