--- /mnt/data/2021_11_multi_chunk_repair/APRConfig/results_defects4j2/defects4j2_JacksonDatabind_87/defects4j2_JacksonDatabind_87/src/main/java/com/fasterxml/jackson/databind/util/StdDateFormat.java	2022-05-03 17:42:01.626299331 -0400
+++ /mnt/data/2021_11_multi_chunk_repair/APRConfig/results_defects4j2/defects4j2_JacksonDatabind_87/mcrepair_pfl/patch/susp_0/donor_23/patch_0/StdDateFormat.java	2022-05-03 18:05:14.084456985 -0400
@@ -436,7 +436,8 @@
          * quite simple because we already set date format timezone to be
          * UTC, and hence can just strip out 'Z' altogether
          */
-        int len = dateStr.length();
+        TimeZone DEFAULT_TIMEZONE = _timezone;
+int len = dateStr.length();
         char c = dateStr.charAt(len-1);
         DateFormat df;
         String formatStr;

