--- /mnt/data/2021_11_multi_chunk_repair/APRConfig/results_defects4j2/defects4j2_Codec_2/defects4j2_Codec_2/src/java/org/apache/commons/codec/binary/Base64.java	2022-05-04 02:14:58.836121557 -0400
+++ /mnt/data/2021_11_multi_chunk_repair/APRConfig/results_defects4j2/defects4j2_Codec_2/mcrepair_sfl/patch/susp_3/donor_44/patch_7/Base64.java	2022-05-04 03:02:56.161928917 -0400
@@ -445,7 +445,9 @@
             }
             if (lineLength > 0) {
                 System.arraycopy(lineSeparator, 0, buf, pos, lineSeparator.length);
-                pos += lineSeparator.length;
+                if (pos > 0)  
+
+pos += lineSeparator.length;
             }
         } else {
             for (int i = 0; i < inAvail; i++) {

